resources/qml/dialogs/ReportMessage.qml

```diff
@@ -0,0 +1,78 @@
+// SPDX-FileCopyrightText: Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+import im.nheko
+
+ApplicationWindow {
+    required property string eventId
+
+    width: 400
+    height: gl.implicitHeight + 2 * Nheko.paddingMedium
+    title: qsTr("Report message")
```

Comment:
I think this is missing an explanation, that only your server admin might see the report, not the mod of the room and most admins don't read those reports anyway.